packages/components/src/components/input-field/input.tsx

```diff
@@ -1,23 +1,59 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { getCurrencyDisplayCode } from '@deriv/shared';
+import { TChangeEvent } from './input-field';
+
+type TInputProps = {
+    ariaLabel: string;
+    changeValue: (e: TChangeEvent, callback?: (evt: TChangeEvent) => void) => void;
+    checked?: number | string;
```

Comment:
as I see this prop 'checked' is used only in stories.
maybe it should be boolean instead of number or string?

Reply:
@yauheni-deriv, `checked` is actually passed into the `input` component, because this `input` can also be a `checkbox`. Please refer to the screenshot below for clarity

<img width="1118" alt="checked" src="https://user-images.githubusercontent.com/104334373/220886550-5436744b-ac64-4e4f-8605-e0ea901de03d.png">

Reply:
yes, I understand it. if it is checkbox, basically 'checked' means true or false, what is the reason to accept other types?
Isn’t it better to make it boolean when it is passed to input-field, than convert it to boolean here.

Reply:
I'm sorry , I'd like to stay with my opinion. I believe if we don’t use property 'checked'  for let say showing it for user as a string or number value, and we just pass it into internal input, it should be boolean. And if outside the input we pass something to this property we should convert it to boolean outside the input component.
the name of the property says explicitly, that this property is boolean.

Reply:
😆 It's fine @yauheni-deriv. I think I just forgot to change this. I'll update it. No worries 😄